dockstore-common/src/main/java/io/dockstore/common/yaml/YamlWorkflow.java

```diff
@@ -40,6 +40,8 @@
     @NotNull
     private String primaryDescriptorPath;
 
+    private Boolean publish;
```

Comment:
Ties with previous comment, why not default false?

Reply:
`null` does nothing
`True` publishes
`False` unpublishes

True and False are intended to correspond with the current API behaviour of publishing and unpublishing.

Imagine a git repo with many branches that we'd like to publish:

- All branches start with null publish (which does nothing).
- Then to publish, we push one of the branches `publish: True`.
- If any feature branches get pushed to, they won't affect this since their publish value is unset.
  + If it were default `false`, then these other branches would unpublish the workflow.

Reply:
Ah, excellent. Was pointed out above too but I still missed it. 
Let's get this in as a code comment here and on squash, make a ticket for documentation since I will definitely forget.